docs/BreakingChanges/074 Assemblies compiled with Regex.CompileToAssembly breaks between 4.0 and 4.5.md

```diff
@@ -0,0 +1,27 @@
+## 74: Assemblies compiled with Regex.CompileToAssembly breaks between 4.0 and 4.5
```

Comment:
{
  "Id": "74",
  "Title": "Assemblies compiled with Regex.CompileToAssembly breaks between 4.0 and 4.5",
  "Details": "If an assembly of compiled regular expressions is built with the .NET Framework 4.5 but targets the .NET Framework 4, attempting to use one of the regular expressions in that assembly on a system with .NET Framework 4 installed throws an exception.",
  "Suggestion": "To work around this problem, you can do either of the following:\n\n- Build the assembly that contains the regular expressions with the .NET Framework 4.\n- Use an interpreted regular expression.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#core",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Text.RegularExpressions.Regex.CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[],System.Reflection.AssemblyName)",
    "M:System.Text.RegularExpressions.Regex.CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[],System.Reflection.AssemblyName,System.Reflection.Emit.CustomAttributeBuilder[])",
    "M:System.Text.RegularExpressions.Regex.CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[],System.Reflection.AssemblyName,System.Reflection.Emit.CustomAttributeBuilder[],System.String)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Text.RegularExpressions.Regex.CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[],System.Reflection.AssemblyName)
- [x] M:System.Text.RegularExpressions.Regex.CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[],System.Reflection.AssemblyName,System.Reflection.Emit.CustomAttributeBuilder[])
- [x] M:System.Text.RegularExpressions.Regex.CompileToAssembly(System.Text.RegularExpressions.RegexCompilationInfo[],System.Reflection.AssemblyName,System.Reflection.Emit.CustomAttributeBuilder[],System.String)